tutorials/eth-api/how-to-build-a-dapp.md

```diff
@@ -236,11 +236,15 @@ Your Hardhat project should already come with a script in the `scripts` folder,
 const hre = require('hardhat');
 
 async function main() {
+  const [deployer] = await hre.ethers.getSigners();
+
   const MintableERC20 = await hre.ethers.getContractFactory('MintableERC20');
-  const token = await MintableERC20.deploy();
-  await token.deployed();
+  const token = await MintableERC20.deploy(deployer.address);
+  await token.waitForDeployment();
 
-  console.log(`Deployed to ${token.address}`);
+  // Get and print the contract address
+  const myContractDeployedAddress = await token.getAddress();
+  console.log(`Deployed to ${myContractDeployedAddress}`);
```

Comment:
closest I could get to the line that needs to be updated.

Line 351 should be updated for ethers v6:

```
      provider = new ethers. BrowserProvider(window.ethereum);
```

Reference: https://docs.ethers.org/v6/migrating/#migrate-providers